### PR TITLE
[TD-11699]<feature>: support gzip

### PR DESCRIPTION
**Pull request by huskar-t**
@sangshuduo @afwerar

### CODE REVIEW

**Review by afwerar — APPROVED** [review]
LGTM

File: config/config.go [hunk 1]
```diff
@@ -87,6 +89,10 @@ func init() {
 	_ = viper.BindEnv("taosConfigDir", "TAOS_ADAPTER_TAOS_CONFIG_FILE")
 	pflag.String("taosConfigDir", "", `load taos client config path. Env "TAOS_ADAPTER_TAOS_CONFIG_FILE"`)
 
+	viper.SetDefault("enableGzip", false)
```
Comment (sangshuduo) [inline config/config.go:92]: default is false

**Review by sangshuduo — COMMENTED** [review]
document need updates too

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**